### DISCUSSION TITLE
Chocolatey Package

### DISCUSSION THREAD

**Discussion by iamLazyCode** [moudey/Shell#147] [General]
Hi Is there any chance to have this great tool as chocolatey package ? 

@moudey

**Comment by moudey** [comment]
> Hi Is there any chance to have this great tool as chocolatey package ?

I will work to support it in the future

**Reply by iamLazyCode** [reply]
Thankyou so much sir that day il be prepared with some more coffees :D

**Comment by moudey** [comment]
https://community.chocolatey.org/packages/nilesoft.shell

**Reply by iamLazyCode** [reply]
> https://community.chocolatey.org/packages/nilesoft.shell

i just cant thankyou enough for your kindness and hard work

![image](https://user-images.githubusercontent.com/13099434/220475939-d57d0c6d-b918-4d12-b0cc-6908edf37d18.png)

**Reply by moudey** [reply]
Thank you for your support

**Reply by iamLazyCode** [reply]
> Thank you for your support

more to come in future :)

**Reply by iamLazyCode** [reply]
> https://community.chocolatey.org/packages/nilesoft.shell

how much does they take to finally approve the package ?

**Reply by moudey** [reply]
I don't know, but there was a false positive on a virus scan and it was corrected

**Reply by iamLazyCode** [reply]
> I don't know, but there was a false positive on a virus scan and it was corrected

How dare they give positive on virus scan to this beauty of a program

**Reply by iamLazyCode** [reply]
> https://community.chocolatey.org/packages/nilesoft.shell

bro i still cant install or find the package.... is there any update when will be the package be passed ?

![image](https://user-images.githubusercontent.com/13099434/221288487-1a562ec8-5687-4c19-8d17-df2c5bff0ef3.png)

**Reply by moudey** [reply]
It's still in review but you can use the following command
```
choco install nilesoft.shell --version=1.8.1
```

**Reply by iamLazyCode** [reply]
> It's still in review but you can use the following command
> 
> ```
> choco install nilesoft.shell --version=1.8.1
> ```

damn are they reviewing global crisis....thankyou sooo much works flawlessly sir thanks

**Comment by iamLazyCode** [comment]
@moudey bro is the package still not approved ? whats the status

**Reply by moudey** [reply]
This is their message response
> Moderators are volunteers, so sometimes the queue can build up a bit. Rest assured that your package will be reviewed soon.

**Reply by iamLazyCode** [reply]
> This is their message response
> 
> > Moderators are volunteers, so sometimes the queue can build up a bit. Rest assured that your package will be reviewed soon.

i am speechless

**Reply by iamLazyCode** [reply]
> This is their message response
> > Moderators are volunteers, so sometimes the queue can build up a bit. Rest assured that your package will be reviewed soon.

Bro there is an update :(

![20230411_134305.jpg](https://user-images.githubusercontent.com/13099434/231098377-b4ea22c8-ac2e-41d1-81f9-b2603eb96fde.jpg)

**Comment by iamLazyCode** [comment]
@moudey i hope this gets released soon sir :D

**Reply by moudey** [reply]
Well as soon as possible I will solve this issue

**Reply by iamLazyCode** [reply]
> Well as soon as possible I will solve this issue

any updates sir ? :D

**Reply by iamLazyCode** [reply]
> Well as soon as possible I will solve this issue

![image](https://github.com/moudey/Shell/assets/13099434/3d84d61b-b977-40fd-b88a-6c7ca82b756d)

**Reply by moudey** [reply]
Please refer to this page https://community.chocolatey.org/packages/nilesoft-shell/1.8.1

**Reply by moudey** [reply]
Has been approved

https://community.chocolatey.org/packages/nilesoft-shell/1.8.1

**Reply by iamLazyCode** [reply]
> Has been approved
> 
> https://community.chocolatey.org/packages/nilesoft-shell/1.8.1
> 
> 

Hahahaha finally before humanity ended they did it...congrats sir
